SPEC.md

```diff
@@ -86,6 +86,7 @@
   * [File write_objects(Array\[Object\])](#file-write_objectsarrayobject)
   * [File write_json(mixed)](#file-write_jsonmixed)
   * [File size(File, \[String\])](#float-sizefile-string)
+  * [File sub(String, String, String)](#string-substring-string-string)
```

Comment:
Oh nice catch. For the link the way I do it is I create the section, generate an html preview and then use whatever link the anchor points to.